### PR TITLE
[refer #PGPRO-4978] Update the .gitignore file.

### PR DESCRIPTION
**Pull request by rzharkov**
tags: ptrack

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 *.so
 *.o
 Dockerfile
+ptrack--2.1.sql
```
Comment (ololobus) [inline .gitignore:5]: `ptrack--2.1.sql` is a base extension sql file, it should not be in the `.gitignore`. Check `Makefile`, I have changed things a bit
```suggestion
```

**Review by ololobus — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (rzharkov) [reply .gitignore:5]: Ok, but now I need to find and recheck an extension which contains a similar sql in its .gitignore file.

**Review by rzharkov — COMMENTED** [review]
What do I need to do now?

**Comment by rzharkov** [comment]
I removed ptrack--2.1.sql from the .gitignore file using git command line and then I rebased the branch.

**Pull request merged**